zipkin-server/README.md

```diff
@@ -114,6 +115,7 @@ zipkin-server is a drop-in replacement for the [scala query service](https://git
     * `STORAGE_TYPE`: SpanStore implementation: one of `mem`, `mysql`, `cassandra`, `elasticsearch`
     * `COLLECTOR_PORT`: Listen port for the scribe thrift api; Defaults to 9410
     * `COLLECTOR_SAMPLE_RATE`: Percentage of traces to retain, defaults to always sample (1.0).
+    * `LOGS_URL`: Logs query service url pattern
```

Comment:
actually, looking at this here, it seems we probably shouldn't have given this a special env variable (since none of the other UI ones do). Can you remove the mapping from zipkin-server-shared.yml? Instead, it would be set normally with `ZIPKIN_UI_LOGS_URL` (which eliminates the need to duplicate some of the docs)